pkg/repository/cache.go

```diff
@@ -109,24 +109,24 @@ func (c *cache) isSubmittedCacheHit(submitted *unstructured.Unstructured, key st
 
 func (c *cache) isPersistedCacheHit(key string, existingObj *unstructured.Unstructured, persistedCached *unstructured.Unstructured) bool {
 	c.logger.Info("considering object", "key", key, "existingName", existingObj.GetName())
-	existingSpec, ok := existingObj.Object["spec"]
+	existingSpec, ok := existingObj.Object["Spec"]
```

Comment:
is this right? I'm not sure how the conversion happens under the hood, but if it's a `map[string]interface{}`, then I believe it's case sensitive (I guess this came in a late refactor?)

Reply:
yeah we're moving too fast, cleaning some strange refactors now.